plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -762,6 +762,38 @@ fu_plugin_unlock (FuPlugin *plugin, FuDevice *device, GError **error)
 	return TRUE;
 }
 
+static void
+fu_plugin_uefi_update_state_notify_cb (GObject *object,
+				       GParamSpec *pspec,
+				       FuPlugin *plugin)
+{
+	FuDevice *device = FU_DEVICE (object);
+	GPtrArray *devices;
+	g_autofree gchar *msg = NULL;
+
+	/* device is not in needs-reboot state */
+	if (fu_device_get_update_state (device) != FWUPD_UPDATE_STATE_NEEDS_REBOOT)
+		return;
+
+	/* only do this on hardware that cannot coalesce multiple capsules */
+	if (!fu_device_has_custom_flag (device, "no-coalesce"))
+		return;
+
+	/* mark every other device for this plugin as non-updatable */
+	msg = g_strdup_printf ("Cannot update as %s [%s] needs reboot",
+			       fu_device_get_name (device),
+			       fu_device_get_id (device));
+	devices = fu_plugin_get_devices (plugin);
+	for (guint i = 0; i < devices->len; i++) {
+		FuDevice *device_tmp = g_ptr_array_index (devices, i);
+		if (device_tmp == device)
+			continue;
+		fu_device_remove_flag (device_tmp, FWUPD_DEVICE_FLAG_UPDATABLE);
+		fu_device_add_flag (device_tmp, FWUPD_DEVICE_FLAG_UPDATABLE_HIDDEN);
+		fu_device_set_update_message (device_tmp, msg);
```

Comment:
shouldn't this be update error?

Reply:
Hmm, I wasn't sure. It's not like the error was attempted and it failed, which I think is how we use update_error now.

Reply:
Update message is shown when an update is done.  It's stuff like the WD19 does where it lets you know the update finishes next time you unplug.  I think of Update Error as any time it's not updatable, or failed to update here is why.

Reply:
Excellent, agreed -- fixed.